website/docs/error-reference.md

```diff
@@ -542,6 +542,17 @@ Ruby `class`. For more, see the docs for
 The `has_attached_class!` annotation cannot be given a contravariant `:in`
 annotation because `T.attached_class` is only allowed in output positions.
 
+## 3515
+
+You are using the chained `sig` syntax however it is malformed. Ensure a block
+is passed and there are no duplicate chained methods. You can only chain
+`abstract`, `final`, `override` and `overridable` methods on a `sig`.
```

Comment:
I think this description is written from an implementor's perspective, and not from what would be most helpful for the user.  (e.g. what does "duplicate chained methods" mean?  the error about duplicates is for the same method being invoked inside and outside of the block, but "duplicate chained methods" sounds like it might be more like `sig.abstract.abstract`)

Reply:
I could modify that sentence a bit. But if we want detailed descriptions we would have to break down the error into multiple smaller ones.